test/api-e2e/api_test.go

```diff
@@ -60,7 +60,7 @@ func (s *APISuite) TestISBSVC() {
 		Expect().
 		Status(200).Body().Raw()
 	assert.Contains(s.T(), getISBSVCBody, fmt.Sprintf(`"name":"%s"`, testISBSVCName))
-	assert.Contains(s.T(), getISBSVCBody, `"status":"healthy"`)
+	assert.Contains(s.T(), getISBSVCBody, `"status":"inactive"`)
```

Comment:
Is it because it's too fast, that controller hasn't get a chance to reconcile?

Reply:
It would be better to have a try with timeout until it's `healthy`.

Reply:
I'll make that change, should I open a smaller PR or revert this one?

Reply:
please open a new one.